src/pages/regulatory/_eu-grid.tsx

```diff
@@ -3,9 +3,10 @@ import styled from 'styled-components'
 import { CssGrid, CssGridColumn } from 'components/containers'
 import { localize, WithIntl, Localize } from 'components/localization'
```

Comment:
```suggestion
import { WithIntl, Localize } from 'components/localization'
```

Please remove the extra imports too :pray: